### PR TITLE
Adds Chris Winters' testimonial to Pittsburgh 2015 index

### PR DESCRIPTION
**Pull request by colindean**
Need a :+1: from one of @sethvargo @cmluciano @littleidea @meatballhat @j41manning @carsomyr

And a :+1: from @cwinters

### CODE REVIEW

File: site/content/events/2015-pittsburgh/index.html [hunk 1]
```diff
@@ -35,4 +35,6 @@
 
 > "I had a lot of fun at DevOps Days Pittsburgh in 2014. I met dozens of passionate, experienced people interested in changing the culture of development an operations. I even hired two of them!" - [Casey West](https://www.twitter.com/caseywest)
 
+> "Devopsdays Pittsburgh 2014, on paper, shouldn't have worked. The stew of people from different backgrounds -- sysadmins, front-end and back-end developers, database administrator, project manager, QA engineer, consultants -- how could these people all get along? But we did, we really really did. The talks were honest and direct, and the open spaces harnessed the chaos of disparate backgrounds into memorable discussions that still resonate over a year later. Recommended if you have anything to do with making or using software." - [Chris Winters](https://www.twitter.com/cwinters)
```
Comment (cmluciano) [inline site/content/events/2015-pittsburgh/index.html]: database administrators

**Comment by cmluciano** [comment]
Minor grammars, but overall I love this review. Thanks @cwinters

**Comment by sethvargo** [comment]
:+1: pending @cwinters

**Comment by cwinters** [comment]
:+1:  anytime

**Pull request merged**